models/MonthlyProfile.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "MonthlyProfile",
+  "type": "object",
+  "description": "Prêt ou plan progressif variant mensuellement.",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Fréquence des variations.",
+      "enum": [
+        "MONTHLY"
+      ]
+    },
+    "variation": {
```

Comment:
Should we really be THAT generic? I mean will we really see negative values?

Reply:
In fact, since the positive values have sense for young people, for which the salary will increase, negative values can have a lot of sense for people close to retirement for instance.